src/table/virtualizedDataTable.jsx

```diff
@@ -140,48 +498,90 @@ class VirtualizedDataTable extends Component {
         },
       });
     };
-  }
 
   // handles the resizable column drag end
-  prvHandleResizeDragStop(evt) {
+  prvHandleResizeDragStop = (evt) => {
+    const { onColumnResizeEndCallback } = this.props;
     const { resizeDrag: { dragColumn, dragStart, origWidth } } = this.state;
     const newColumnWidth = origWidth + (evt.clientX - dragStart);
     if (newColumnWidth > 0) {
-      this.props.onColumnResizeEndCallback(newColumnWidth, dragColumn);
+      onColumnResizeEndCallback(newColumnWidth, dragColumn);
     }
 
+    this.prvRecomputeTables();
+
     this.setState({
       resizeDrag: null,
       recentDragEnd: dragColumn,
     });
-  }
+  };
+
+  // handles mouse wheel.
+  prvHandleWheel = (evt) => {
+    if (this.prvCellGridDOM) {
+      evt.preventDefault();
+
+      const { wheelDelta } = this.props;
+      const { state: { scrollTop, scrollLeft } } = this.prvCellGridRef;
+      const delta = evt.deltaY || evt.detail || evt.wheelDelta;
+      const newScrollTop = Math.max(0, scrollTop + ((delta > 0) ? wheelDelta : -wheelDelta));
+      this.prvCellGridDOM.scrollTop = newScrollTop;
+
```

Comment:
Force a scroll event on wheel, which will be handled by our scroll handler.   Disallow automatic handling of the mouse wheel in favor of always using the scroll handler, which will keep the tables in unison